githubscript/docker.d/worker.yml

```diff
@@ -63,6 +69,6 @@ github_projects:
 taskcluster_scope_prefixes:
   $switch:
     'COT_PRODUCT == "mobile"':
-       ["project:mobile:fenix:releng:github:", "project:mobile:android-components:releng:github:"]
+       ["project:mobile:fenix:releng:github:", "project:mobile:android-components:releng:github:", "project:mobile:focus-android:releng:github:"]
```

Comment:
Nit: I may be wrong but we may need to whitelist this scope in https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L356 CC @escapewindow

Reply:
Interesting, I didn't add the "project:mobile:focus-android:releng:googleplay:product:" to that module when modifying the pushapk scriptworker. Is this specific to the githubscript?

Reply:
@sarah-clements ideally we make all production-only scopes restricted there, so if we missed pushapk, let's add it.

e.g., we have

```
# scope                                                 repo/branch alias
"project:mobile:fenix:releng:googleplay:product:fenix": "fenix-repo",
```

[here](https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L360) to limit pushing the fenix apk from the fenix repo itself, and https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L356 for limiting the a-c github scope to the a-c repo as @MihaiTabara mentioned.

Reply:
Made changes in scriptworker for this: https://github.com/mozilla-releng/scriptworker/pull/522